setup.py

```diff
@@ -65,14 +72,26 @@
     name='pyq',
     packages=['pyq', 'pyq.tests', ],
     package_dir={'': 'src'},
-    qlib_scripts=['python.q', 'p.k', 'pyq-operators.q'],
+    qlib_scripts=['python.q', 'p.k', 'pyq-operators.q', '../../embedPy/p.q'],
     ext_modules=[
         Extension('pyq._k', sources=['src/pyq/_k.c', ],
                   extra_compile_args=CFLAGS),
     ],
     qext_modules=[
-        Extension('p', sources=['src/pyq/p.c', ],
+        Extension('pyq', sources=['src/pyq/pyq.c', ],
                   extra_compile_args=CFLAGS),
+        Extension('p', sources=['embedPy/p.c', ],
+                  extra_compile_args=CFLAGS + [
+                      '-Isrc/pyq/kx',
+                      '-I' + sysconfig.get_python_inc(),
+                      '-I' + numpy.get_include(),
+                      '-DPH=L"%s:%s"' % (sys.prefix, sys.exec_prefix),
+                      '-DDY="%s"' % sysconfig.get_config_var('LDLIBRARY'),
```

Comment:
LDLIBRARY is a directory on brew!

Reply:
.. but DY is not used on macOS, so we are good!